dockerfiles/theia/Dockerfile

```diff
@@ -107,6 +107,8 @@ EXPOSE 3100 3130
 
 COPY --from=builder /home/theia-dev/theia-source-code/production/plugins /default-theia-plugins
 
+RUN wget -P /default-theia-plugins https://github.com/che-incubator/vscode-git/releases/download/1.30.1/vscode-git-1.3.0.1.vsix
+
```

Comment:
maybe put that next to https://github.com/eclipse/che-theia/blob/ffbef321470fff581c1fd78a45b1decf4124ecc5/dockerfiles/theia/Dockerfile#L125

Reply:
done